src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -128,62 +136,79 @@ class Container extends React.Component<Props, ComponentState> {
     }
 
     UNSAFE_componentWillReceiveProps(nextProps: Props) {
-        if (nextProps.open !== this.props.open) {
-            // Build entity options based on current model locale
-            const currentAppLocale = nextProps.app.locale
-            const preBuiltLocale = PreBuiltEntities.find(entitiesList => entitiesList.locale === currentAppLocale)
-            if (!preBuiltLocale) {
-                throw new Error(`Could not find locale: ${currentAppLocale} within list of supported locales: ${PreBuiltEntities.map(e => e.locale).join(', ')}`)
-            }
+        if (nextProps.open) {
```

Comment:
Looks like a lot of changes but it was mostly indentation / white-space from splitting this `if` statement.  Previously this only allowed running when the modal was opened, but conditions have to be recomputed while the model is open.